src/TransformUtils.ts

```diff
@@ -5,6 +5,11 @@ import * as TS from 'typescript';
 // inside the ConfigSet right
 export interface ConfigSet {
   compilerModule: typeof TS;
```

Comment:
if the `const ts = configSet.compilerModule` above is redundant and can be removed, this `compilerModule` can be removed too. In the latest version of `ts-jest`, this property is no longer exposed as public typings.

Reply:
Do you mean by this that `configSet.compilerModule` equals `configSet.tsCompiler`?

Reply:
actually those return 2 different things, `compilerModule` returns the current typescript package of the project while `tsCompiler` returns the compiler instance created internally in `ts-jest`. 

But in `AngularConstructorParametersTransformers.ts`, `compilerModule` isn't used anywhere so I think this property `compilerModule` can be removed.

Reply:
In the other transformers we still need functionality like `configSet.compilerModule.getMutableClone(node)`, which was usually injected by the `compilerModule`, being an instance of `typescript`.

Reply:
I see, so I think `ts-jest` should expose that one again in public typings 👍

Reply:
Yes, this would be great: `compilerModule: typeof TypeScript` where `import * as TypeScript from 'typescript'`

Reply:
will be like this

```
export declare class ConfigSet {
    readonly parentOptions?: TsJestGlobalOptions | undefined;
    get versions(): Record<string, string>;
    get compilerModule(): TTypeScript;
    get tsCompiler(): TsCompiler;
    get tsJestDigest(): string;
    readonly logger: Logger;
    constructor(jestConfig: Config.ProjectConfig, parentOptions?: TsJestGlobalOptions | undefined, parentLogger?: Logger);
}
```

where `TTypeScript` is
```
import * as _ts from 'typescript';
export declare type TTypeScript = typeof _ts;
```

It was public before but then marked as internal 😃

Reply:
If you assume, that only `ts-jest` authors transformers, it is correctly marked as internal :smile:

Reply:
For now I will extend the type with the `compilerModule` inline. Let me know when there's a new version out with the public typings!

Reply:
FYI, `ts-jest` 26.1.2 released. It included the exposed typing for `compileModule`.